sale_company_currency/views/sale_order_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <!-- SO -->
+    <record id="view_order_tree" model="ir.ui.view">
+      <field name="name">sale.order.tree</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale.view_order_tree"/>
+      <field name="arch" type="xml">
+
+        <!-- Remove multicurrencies sums (makes no sense)-->
```

Comment:
@leemannd 
I have been working in trading many years and I do need to see the list in the currency of the PO and I dont care about my accounting currency. Removing the sum makes sense but removes as well a potential useful feature ("How much do I have to pay in USD?")

Will not block for it though.